build.gradle.kts

```diff
@@ -68,8 +69,8 @@ subprojects {
             name = "github"
             url = uri("https://maven.pkg.github.com/gitliveapp/packages")
             credentials {
-                username = project.findProperty("gpr.user") as String? ?: System.getenv("gpr.user")
-                password = project.findProperty("gpr.key") as String? ?: System.getenv("gpr.key")
+                username = project.findProperty("gpr.user") as String? ?: System.getenv("USERNAME")
+                password = project.findProperty("gpr.key") as String? ?: System.getenv("TOKEN")
```

Comment:
this is only for the private maven github packages, so belongs here and not in master.